### PR TITLE
Add Ternary oprator guidelines

### PR DESCRIPTION
**Pull request by brendt**

### CODE REVIEW

File: content/code-style/laravel-php.md [hunk 1]
```diff
@@ -75,6 +75,24 @@ Always use fully qualified class names in docblocks.
  */
 ```
 
+## Ternary operators
+
+Use multi-line ternay operators unless it's a really short expression.
```
Comment (freekmurze) [inline content/code-style/laravel-php.md]: `ternay` should be `ternary`

File: content/code-style/laravel-php.md [hunk 2]
```diff
@@ -75,6 +75,24 @@ Always use fully qualified class names in docblocks.
  */
 ```
 
+## Ternary operators
```
Comment (freekmurze) [inline content/code-style/laravel-php.md]: Also add a link to this anchor at the top of the page.

File: content/code-style/laravel-php.md [hunk 3]
```diff
@@ -75,6 +75,24 @@ Always use fully qualified class names in docblocks.
...
+
+Use multi-line ternay operators unless it's a really short expression.
+
+```php
+// Good
+
+$result = $flag && $object instanceof Model
```
Comment (freekmurze) [inline content/code-style/laravel-php.md]: I don't like the fact that this example has `&&` in it. I know it's subjective, but I feel that it makes the example kinda "heavy".

**Comment by brendt** [comment]
@freekmurze I've pushed the changes addressing your feedback.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**